### ISSUE TITLE
Clarification Needed on Running Command for Single Depot Executable

### ISSUE THREAD

**Issue by nsmstn** [pengfeihe-angers/minmax-mTSP#1]
When I extracted the executable for the single depot from `minmx mTSP.rar` and ran it as described in `readme.txt` using `./ma mtsp150_3.txt 100 -1 1`, I encountered the message: "the input parameters in configuration are wrong."

However, it worked correctly when I removed the final "1" and ran `./ma mtsp150_3.txt 100 -1`.

Is the correct running command `./ma <input file> <running time> <seed>`?

Thank you for clarifying.

**Comment by pengfeihe-angers** [comment]
Yes, Thank you very much. I have revised the readme in this project.
